### ISSUE TITLE
Problems

### ISSUE THREAD

**Issue by Demetryi** [avocode/avocode#2203]
---
* **Version**: 3.6.0
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 436158
---
```
TypeError: Cannot read property 'set' of undefined
    at file:///C:/Users/dev/AppData/Local/avocode/app-3.6.0/resources/app.asar/dist/avocode.min.js:1:4052319
    at updateInDeepMap (C:\Users\dev\AppData\Local\avocode\app-3.6.0\resources\app.asar\node_modules\immutable\dist\immutable.js:1973:22)
    at updateInDeepMap (C:\Users\dev\AppData\Local\avocode\app-3.6.0\resources\app.asar\node_modules\immutable\dist\immutable.js:1982:23)
    at updateInDeepMap (C:\Users\dev\AppData\Local\avocode\app-3.6.0\resources\app.asar\node_modules\immutable\dist\immutable.js:1982:23)
    at Map.updateIn (C:\Users\dev\AppData\Local\avocode\app-3.6.0\resources\app.asar\node_modules\immutable\dist\immutable.js:1280:26)
    at file:///C:/Users/dev/AppData/Local/avocode/app-3.6.0/resources/app.asar/dist/avocode.min.js:1:4052283
    at C:\Users\dev\AppData\Local\avocode\app-3.6.0\resources\app.asar\node_modules\immutable\dist\immutable.js:3907:58
    at C:\Users\dev\AppData\Local\avocode\app-3.6.0\resources\
```

**Comment by crashmajstr** [comment]
Hi @Demetryi  , we're currently working on a fix, I am sorry for the inconvenience, we'll let you know once it's fixed. 
Closing with respect to #2202 
Best,
Josef

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ll release the fix in an upcoming hour or so. It’ll be version 3.6.1.
For now, you can just download and use our older stable build 3.5.6 

Download links
Mac: https://we.tl/t-bS4jyZufpT
Win: https://we.tl/t-12nDKmpafd
Linux: https://we.tl/t-krrbw7ZRvB

I’ll let you know once the fix is out. Sorry for the inconvenience.

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ve just released the new version of Avocode 3.6.1 containing the fix for this issue. 
Please, update your desktop app, if you use the web app the new version is already there.
Hope everything will be ok from now on.
I am very sorry for the inconvenience. 
On behalf of Avocode,
Josef Kettner
